bears/general/KeywordBear.py

```diff
@@ -79,22 +79,55 @@ def run(self,
             filename,
             file,
             keywords: list=['todo', 'fixme'],
+            regex_keyword: str='',
             dependency_results: dict=None):
         '''
         Checks the code files for given keywords.
 
         :param keywords:
             A list of keywords to search for (case insensitive).
             Default are TODO and FIXME.
+        :param regex_keyword:
+            A regular expression to search for matching keywords in a file.
         '''
         keywords_regex = re.compile(
             '(' + '|'.join(re.escape(key) for key in keywords) + ')',
             re.IGNORECASE)
 
+        regex = re.compile(regex_keyword)
+
         comments = _get_comments(dependency_results)
 
+        message = "The line contains the keyword '{}'."
+        yield from self.check_keywords(filename, file, comments,
+                                       keywords_regex, message)
```

Comment:
@sils but that is already handled... even if the keywords setting is empty by default we are checking for TODO|FIXME (case insensitive). IMO adding an ```if``` won't make sense because keywords setting is already getting converted to to regex for checking todo and fixme.

However this is not the case we are calling ```check_keywords``` for the second time. so ```if```` condition there makes sense.